processing/src/main/java/io/druid/query/filter/DruidLongPredicate.java

```diff
@@ -28,5 +28,25 @@
 
```

Comment:
The class-level Javadoc comment is stale because now the Druid's interface has two methods.

Reply:
Please add "Note: this is not a `{@link PublicApi}` or an `{@link ExtensionPoint}` of Druid." to the class-level Javadoc comment.

Reply:
done. removed stale comment and added note.